man/marked.1.txt

```diff
@@ -56,7 +56,7 @@ OPTIONS
        --smart-lists
 	      Use smarter list behavior than the original markdown.
 
-       --lang-prefix [prefix]
+       --language-prefix [prefix]
```

Comment:
this parameter should still be `--lang-prefix` since that is what the option is called